packages/server/src/models/game/game.model.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class Game {
+    @Field(_type => ID)
+    id: number;
+    /**
+     * The date it was created
+     */
+    creationDate: Date
+}
```

Comment:
You should definitely setup your editor to automatically add new lines at EOF 😛 

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

```
"files.insertFinalNewline": true
```

Reply:
Btw, we should probably try to add a linting rule for that

Reply:
Fixed with linter and editorconfig